### PR TITLE
updated Assertj version

### PR DESCRIPTION
**Pull request by vigneshpushkaranHMCTS**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: dependency-check-suppressions.xml [hunk 1]
```diff
@@ -110,4 +110,15 @@
         <notes>cpe:2.3:a:google:guava:27.0 suppression, root cause and fix needs to be carried out separately</notes>
         <cve>CVE-2020-8908</cve>
     </suppress>
+    <suppress>
```
Comment (amitchidrewar1301) [inline dependency-check-suppressions.xml:113]: Add the until tag with the expected date,  also as discussed please update the notes tag with the reason for adding suppression

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java [hunk 2]
```diff
@@ -137,11 +137,12 @@ public void searchAllPaymentsWithCcdCaseNumberShouldReturnRequiredFieldsForVisua
 
         assertThat(payment.getCcdCaseNumber()).isEqualTo("ccdCaseNumber1");
 
-        assertThat(payment.getReference()).isNotBlank();
```
Comment (AyanDuttaHMCTS) [inline api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java:140]: Please try to keep the same test case logic with new Library code

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java [hunk 3]
```diff
@@ -137,11 +137,12 @@ public void searchAllPaymentsWithCcdCaseNumberShouldReturnRequiredFieldsForVisua
...
+        assertThat(payment.getChannel()).isNotBlank();
         assertThat(payment.getAmount()).isPositive();
         assertThat(payment.getDateCreated()).isNotNull();
-        assertThat(payment.getCustomerReference()).isNotBlank();
```
Comment (AyanDuttaHMCTS) [inline api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java:143]: Please try to keep the same test case logic with new Library code

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/PaymentControllerTest.java [hunk 4]
```diff
@@ -380,10 +380,12 @@ public void searchAllPayments_withCcdCaseNumber_shouldReturnRequiredFieldsForVis
 
         assertThat(payment.getCcdCaseNumber()).isEqualTo("ccdCaseNumber1");
 
-        assertThat(payment.getReference()).isNotBlank();
```
Comment (AyanDuttaHMCTS) [inline api/src/test/java/uk/gov/hmcts/payment/api/componenttests/PaymentControllerTest.java:383]: Please try to keep the same test case logic with new Library code

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/PaymentControllerTest.java [hunk 5]
```diff
@@ -380,10 +380,12 @@ public void searchAllPayments_withCcdCaseNumber_shouldReturnRequiredFieldsForVis
...
+        assertThat(payment.getChannel()).isNotBlank();
         assertThat(payment.getAmount()).isPositive();
         assertThat(payment.getDateCreated()).isNotNull();
-        assertThat(payment.getCustomerReference()).isNotBlank();
```
Comment (AyanDuttaHMCTS) [inline api/src/test/java/uk/gov/hmcts/payment/api/componenttests/PaymentControllerTest.java:386]: Please try to keep the same test case logic with new Library code

Comment (vigneshpushkaranHMCTS) [reply api/src/test/java/uk/gov/hmcts/payment/api/componenttests/PaymentControllerTest.java]: Reverted to old library version

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/PaymentControllerTest.java (hunk 4 above)
Comment (vigneshpushkaranHMCTS) [reply api/src/test/java/uk/gov/hmcts/payment/api/componenttests/PaymentControllerTest.java]: Reverted to old library version

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java (hunk 3 above)
Comment (vigneshpushkaranHMCTS) [reply api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java]: Reverted to old library version

File: api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java (hunk 2 above)
Comment (vigneshpushkaranHMCTS) [reply api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CaseControllerTest.java]: Reverted to old library version

File: dependency-check-suppressions.xml (hunk 1 above)
Comment (vigneshpushkaranHMCTS) [reply dependency-check-suppressions.xml]: Added with expected date

**Review by amitchidrewar1301 — APPROVED** [review]

**Review by aravindan89 — APPROVED** [review]

**Comment by vigneshpushkaranHMCTS** [comment]
Duplcate PR -  closed

**Pull request closed without merging**